.vscode/settings.json

```diff
@@ -1,4 +1,7 @@
 {
+  "[javascript]": {
+    "editor.defaultFormatter": "esbenp.prettier-vscode"
```

Comment:
I don't think this was meant to hit the PR

Reply:
fixed in 7ab324e3a57411239fdfec94d09b11f388c3c97f